src/main/java/seedu/address/logic/commands/EditRecordCommand.java

```diff
@@ -216,16 +242,20 @@ public boolean equals(Object other) {
             }
 
             EditRecordCommand.EditRecordDescriptor otherEditRecordDescriptor =
-                    (EditRecordCommand.EditRecordDescriptor) other;
+                (EditRecordCommand.EditRecordDescriptor) other;
             return Objects.equals(dateTime, otherEditRecordDescriptor.dateTime)
-                    && Objects.equals(conditions, otherEditRecordDescriptor.conditions);
+                    && Objects.equals(conditions, otherEditRecordDescriptor.conditions)
+                    && Objects.equals(filePath, otherEditRecordDescriptor.filePath)
+                    && Objects.equals(patientIndex, otherEditRecordDescriptor.patientIndex);
         }
 
         @Override
         public String toString() {
             return new ToStringBuilder(this)
                     .add("dateTime", dateTime)
+                    .add("filePath", filePath)
                     .add("conditions", conditions)
+                    .add("patientIndex", patientIndex)
                     .toString();
```

Comment:
Is this a consistent format across all implementaation?
dateTime followed by filepath followed by conditions then patientIndex?